wazo_dird/plugins/config_service/plugin.py

```diff
@@ -140,16 +141,39 @@ def load(self, dependencies):
 
 
 class Service:
+    # Changing root logger log-level requires application-wide lock.
+    # This lock will be shared across all instances.
+    _lock = threading.Lock()
+
     def __init__(self, config, bus, controller):
         self._bus = bus
         self._config = config
         self._controller = controller
 
         self._bus.subscribe(TenantCreatedEvent.name, self._on_new_tenant_event)
 
-    def get_current_config(self):
+    def get_config(self):
         return self._config
 
+    def update_config(self, config: dict) -> None:
```

Comment:
Since you're introducing typing in wazo-dird you should add a check in the `.pre-commit-config.yaml` file to be sure your typing is correct and that it stays that way.

Reply:
Thank you, It is done.